frontend/public/components/catalog-item-details.jsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {Modal} from 'patternfly-react/dist/esm/components/Modal';
+import {CatalogItemHeader} from 'patternfly-react-extensions/dist/esm/components/CatalogItemHeader';
+import {PropertiesSidePanel, PropertyItem} from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel';
+
+import {normalizeIconClass} from './catalog-item-icon';
+import {ClusterServicePlanModel} from '../models';
+import {k8sGet} from '../module/k8s';
+import {Timestamp} from './utils';
+
+export class CatalogTileDetails extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      plans: [],
+    };
+  }
+
+  componentDidMount() {
+    const { obj } = this.props.item;
+    this.getPlans(obj);
+  }
+
+  getPlans(obj) {
+    k8sGet(ClusterServicePlanModel, null, null, {queryParams: { fieldSelector: `spec.clusterServiceClassRef.name=${obj.metadata.name}`} })
+      .then(({items: plans}) => {
+        this.setState({
+          plans: _.orderBy(plans, ['spec.externalMetadata.displayName', 'metadata.name'])
+        });
+      });
+  }
+
+  render () {
+    const { closeOverlay } = this.props;
+    const { obj, tileName, tileImgUrl, tileIconClass, tileProvider, tileDescription, href } = this.props.item;
+    const vendor = tileProvider ? `Provided by ${tileProvider}` : null;
+    const iconClass = tileIconClass ? normalizeIconClass(tileIconClass) : null;
+    const supportUrl = _.get(obj, 'spec.externalMetadata.supportUrl');
+    const creationTimestamp = _.get(obj, 'metadata.creationTimestamp');
+    const longDescription = _.get(obj, 'spec.externalMetadata.longDescription');
+    const builderImageTag = _.head(_.get(obj,'spec.tags'));
+    const sampleRepo = _.get(builderImageTag, 'annotations.sampleRepo');
+    const documentationUrl = _.get(obj, 'spec.externalMetadata.documentationUrl');
```

Comment:
Some of these paths are different depending on whether this is a service class and a builder image. We might want separate details components for each. Or we could put these values in the catalog item that is built when we fetch the content, then pass the items to the details pane.